src/test/java/stanfordnlp/corenlp/TextControllerTest.java

```diff
@@ -0,0 +1,59 @@
+package stanfordnlp.corenlp;
+
+import io.github.ardoco.textproviderjson.converter.JsonConverter;
+import io.github.ardoco.textproviderjson.dto.TextDto;
+import io.github.ardoco.textproviderjson.error.InvalidJsonException;
+import org.junit.ClassRule;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.PullPolicy;
+import stanfordnlp.TestUtil;
+
+import java.io.IOException;
+
+public class TextControllerTest {
+
+
+    String username = "defaultadmin";
+    String password = "defaultpassword";
+    String textEndpoint = "/stanfordnlp";
+    static String address;
+    static TestRestTemplate restTemplate;
+
+    @ClassRule
+    public static GenericContainer simpleWebServer = new GenericContainer("lauraxrb/stanfordnlp-microservice")
```

Comment:
```suggestion
    public static GenericContainer simpleWebServer = new GenericContainer("stanfordnlp-microservice")
```

I agree to the comment :)